vic/drivers/classic/src/vic_classic.c

```diff
@@ -247,6 +248,13 @@ main(int   argc,
             ******************************************/
 
             for (rec = startrec; rec < global_param.nrecs; rec++) {
+
+                // Set global reference string (for debugging inside vic_run)
+                sprint_dmy(dmy_str, &(dmy[rec]));
```

Comment:
The PET output from VIC in peril river is conspicuous increasing, it is obviously unreasonable because the sunshine hours is decreasing in those years. And we don't have any data of radiation.


来自 魅族 MX4 Pro